core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/arrayfunction/ArraySymmetricDifferenceExpression.kt

```diff
@@ -0,0 +1,36 @@
+package ch.ergon.dope.resolvable.expression.unaliased.type.arrayfunction
+
+import ch.ergon.dope.resolvable.expression.TypeExpression
+import ch.ergon.dope.validtype.ArrayType
+import ch.ergon.dope.validtype.ValidType
+
+class ArraySymmetricDifferenceExpression<T : ValidType>(
+    firstArray: TypeExpression<ArrayType<T>>,
+    secondArray: TypeExpression<ArrayType<T>>,
+    vararg additionalArrays: TypeExpression<ArrayType<T>>,
+) : ArrayFunctionExpression<T>("ARRAY_SYMDIFF", firstArray, secondArray, *additionalArrays)
+
+fun <T : ValidType> arraySymDiff(
+    firstArray: TypeExpression<ArrayType<T>>,
+    secondArray: TypeExpression<ArrayType<T>>,
+    vararg additionalArrays: TypeExpression<ArrayType<T>>,
+) = ArraySymmetricDifferenceExpression(firstArray, secondArray, *additionalArrays)
+
+// only an alias (defined by couchbase)
```

Comment:
Well, the couchbase-function _ARRAY_SYMDIFF1_ is an alias of the couchabse-function _ARRAY_SYMDIFF_, but in dope apparently we never create the _ARRAY_SYMDIFF1_-String. We should add a ArraySymmetricDifference1Expression, if we want to support that function.
And that symdiff1 is an alias of something is also not relevant in the perspective of dope: We just allow the creation of the couchbase-functions and do not care about the actual implementation (just the parameters and the return value)

Reply:
I changed the two instances of aliased functions, so now SYMDIFF1 is possible.